common/scripted_triggers/wc_maa_triggers.txt

```diff
@@ -103,12 +109,49 @@ like_elekks_trigger = {
 		has_innovation = innovation_elephantry
 	}
 }
+like_dragonhawk_trigger = {
```

Comment:
I would say all centaur units should have bonuses in deserts, steppes, drylands.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/157847758-fc821c20-b4b1-4164-bec3-f09cefd6b548.png)

</details>